lib/node_modules/@stdlib/math/base/special/gcdf/package.json

```
@@ -76,9 +76,9 @@
     "scaffold": {
       "$schema": "math/base@v1.0",
       "base_alias": "gcd",
-      "alias": "gcd",
-      "pkg_desc": "compute the greatest common divisor (gcd)",
-      "desc": "computes the greatest common divisor (gcd)",
+      "alias": "gcdf",
+      "pkg_desc": "compute the greatest common divisor (gcd) of two single-precision floating-point numbers",
+      "desc": "computes the greatest common divisor (gcd) of two single-precision floating-point numbers",
```

```suggestion
      "pkg_desc": "compute the greatest common divisor (gcd)",
      "desc": "computes the greatest common divisor (gcd)",
```

We can leave off the input data types here, as we don't know the exact context in which a scaffold might invoke this function.